carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedWebElement.java

```diff
@@ -1274,28 +1276,38 @@ public boolean isVisible(long timeout) {
     public ExtendedWebElement format(Object... objects) {
         String locator = by.toString();
         By by = null;
+
         if (locator.startsWith("By.id: ")) {
             by = By.id(String.format(StringUtils.remove(locator, "By.id: "), objects));
         }
+
         if (locator.startsWith("By.name: ")) {
             by = By.name(String.format(StringUtils.remove(locator, "By.name: "), objects));
         }
+
         if (locator.startsWith("By.xpath: ")) {
             by = By.xpath(String.format(StringUtils.remove(locator, "By.xpath: "), objects));
         }
         if (locator.startsWith("linkText: ")) {
             by = By.linkText(String.format(StringUtils.remove(locator, "linkText: "), objects));
         }
+
         if (locator.startsWith("partialLinkText: ")) {
             by = By.linkText(String.format(StringUtils.remove(locator, "partialLinkText: "), objects));
         }
+
         if (locator.startsWith("css: ")) {
             by = By.cssSelector(String.format(StringUtils.remove(locator, "css: "), objects));
         }
+
         if (locator.startsWith("tagName: ")) {
             by = By.tagName(String.format(StringUtils.remove(locator, "tagName: "), objects));
         }
 
+        if (locator.startsWith("By.xpath: **")) {
```

Comment:
please add comment with example that this usecase is used for classChain elements declaration